### DISCUSSION TITLE
【求助】谁家小孩

### DISCUSSION THREAD

**Discussion by wushanef** [share121/inter-knot#140] [其他]
今天在十四分街见到一个小孩，有人知道是谁家的吗
![d59bbcab82f8cf2a7049977884e81f1238249ca8f5c2c95e2e172515582dfb7b 0](https://github.com/user-attachments/assets/d4661877-adbf-4486-8921-2f9664c02529)

**Comment by koyki233** [comment]
不道啊，我住十三分街的

**Comment by wum2008** [comment]
我家的
![桃香](https://github.com/user-attachments/assets/d27c2f4b-8854-4345-8f9f-f66b0303331b)

**Comment by Tragedysaver1024** [comment]
这不就是你自己吗（恼）耍我啊kora

**Reply by wushanef** [reply]
唉嘿

**Comment by DeepslateQAQ** [comment]
@wum2008 你6你6

**Comment by wushanef** [comment]
test

**Comment by goodegg8** [comment]
是我们有刺无刺的不登校，麻烦帮忙带到mmk的房间里进行sox活动 ~~~~~~~

**Reply by wushanef** [reply]
好的
![IMG_20240802_160529](https://github.com/user-attachments/assets/30a7573a-4ec1-45d7-9ea6-b958956ce112)

**Comment by llxcx4** [comment]
![Uploading 1722353712532.gif…]()
不道啊

**Comment by Nina123973** [comment]
我家的

**Comment by inderya** [comment]
hhhhhhh

**Comment by kaijinti** [comment]
不知道是谁家的，但我看见过她在大街上好像是在耍流星锤？耍的那叫一个虎虎生风

**Comment by su2569** [comment]
我不道啊

**Comment by Qiling-XwX** [comment]
我家的
![265DEC65ECE486B2D1EA4F8D1FBACC84](https://github.com/user-attachments/assets/1b30f28d-19b2-423a-8c7a-20403973f502)

**Comment by Gmadelyn7** [comment]
我家我家我家的
